connectors/casa/main.py

```diff
@@ -1,7 +1,10 @@
+import logging
 from typing import Any, Dict
 
 import requests
 
+logging.basicConfig(format='[%(asctime)s] {%(filename)s:%(lineno)d} %(levelname)s - %(message)s')
```

Comment:
Outside the scope of this PR, should we have some sort of central config/definition of logging rules in the root directory so that we can apply the same settings across all connectors?

Reply:
Yeah maybe like a util function or something could be good since the connectors run fairly independently, no strong opinions tbh. Might be useful if we have more custom connectors (and more people working on connectors they didn't write)

Reply:
yeah, I think so. It would be nice just to make sure logging is consistent across all connectors we create IMHO. But as I mentioned, it's outside the scope of this PR. Will mark this conversation as resolved, but before I do @scholtzan do you think it makes sense to create an issue for this?

Reply:
Sure, feel free to open an issue. The main problem with sharing is that only the code that is in the connector directory will be available to the Cloud Function. So all the files in the root directory will be missing.